### PR TITLE
fix-28-questions_being_repeated_during_every_round

### LINKED ISSUE #28: 🐛 Bug Report: Questions being repeated during every round
### 👟 Reproduction steps

Start using the app and play the round, you will notice that sometimes the questions asked before are being repeated.

### 👍 Expected behavior

It should not repeat the question

### 👎 Actual Behavior

It actually repeats the question

### 🎲 Appwrite version

Different version (specify in environment)

### 💻 Operating system

Something else

### 🧱 Your Environment

Flutter version: 3.3.4
Dart version: 2.18.2 (stable) (Tue Sep 27 13:24:11 2022 +0200) on "linux_x64"
Appwrite package version: 8.1.0

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by SagittariusA11**
Have replaced the repeated question in quiz_app_questions.json.

## What does this PR do?

In this PR I have just changed a repeated question in quiz_app_questions.json file.

## Test Plan

Didn't required any test as have just changed the question in string format which results in no effective execution change.

## Related PRs and Issues

No

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: quiz_app_questions.json [hunk 1]
```diff
@@ -35,9 +35,9 @@
 		"answer": "Motherboard"
 	},
 	{
-		"question": "Which of these is not a peripheral, in computer terms?",
-		"options": [ "Keyboard", "Mouse", "Motherboard", "Monitor" ],
-		"answer": "Motherboard"
+		"question": "The first alpha version of Flutter was released in ?",
```
Comment (stnguyen90) [inline quiz_app_questions.json:38]: Would you please remove the extra space?

```suggestion
		"question": "The first alpha version of Flutter was released in?",
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Thanke for the PR! 🙏🏼 Just one comment

File: quiz_app_questions.json (hunk 1 above)
Comment (SagittariusA11) [reply quiz_app_questions.json]: Have removed the space. Sorry for the mistake.

File: quiz_app_questions.json [hunk 2]
```diff
@@ -35,9 +35,9 @@
 		"answer": "Motherboard"
 	},
 	{
-		"question": "Which of these is not a peripheral, in computer terms?",
-		"options": [ "Keyboard", "Mouse", "Motherboard", "Monitor" ],
-		"answer": "Motherboard"
+		"question": "The first alpha version of Flutter was released in?",
```
Comment (stnguyen90) [inline quiz_app_questions.json:38]: Womp! Sorry for the multiple updates. Grammar-wise and to match the rest of the questions, it would be better to reword this to:

```suggestion
		"question": "When was the first alpha version of Flutter released?",
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: quiz_app_questions.json (hunk 2 above)
Comment (SagittariusA11) [reply quiz_app_questions.json]: Yes, this is better version. Thankyou.

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!
